pkg/clusterinit/onboard/certmanager/certmanager.go

```diff
@@ -165,18 +165,19 @@ loop:
 }
 
 // queryRedHatCatalog pull information regarding the cart-manager package from the Red Hat catalog.
-func queryRedHatCatalog(ctx context.Context, clientConnFactory GRPCClientConnFactory, port string) (pack *Package, err error) {
+func queryRedHatCatalog(ctx context.Context, clientConnFactory GRPCClientConnFactory, port string) (pack *Package, retErr error) {
+	pack = &Package{}
```

Comment:
Sorry for the back and forth but, is this required?

Reply:
This is the "real" change on this PR, after this that `invalid memory address or nil pointer dereference` was solved.
The `err/retErr` is just to make the code more clear.